osquery/tables/other/yara_utils.cpp

```diff
@@ -208,11 +208,38 @@ int YARACallback(int message, void *message_data, void *user_data) {
   if (message == CALLBACK_MSG_RULE_MATCHING) {
     Row *r = (Row *) user_data;
     YR_RULE *rule = (YR_RULE *) message_data;
+
     if ((*r)["matches"].length() > 0) {
       (*r)["matches"] += "," + std::string(rule->identifier);
     } else {
       (*r)["matches"] = std::string(rule->identifier);
     }
+
+    YR_STRING *string;
+    yr_rule_strings_foreach(rule, string) {
+      YR_MATCH *match;
+      yr_string_matches_foreach(string, match) {
+        if ((*r)["strings"].length() > 0) {
+          (*r)["strings"] += "," + std::string(string->identifier);
+        } else {
+          (*r)["strings"] = std::string(string->identifier);
+        }
+
+        std::stringstream ss;
+        ss << std::hex << (match->base + match->offset);
+        (*r)["strings"] += ":" + ss.str();
```

Comment:
Yes, YARA outputs offsets in hex (https://github.com/plusvic/yara/blob/master/yara.c#L577). It also will print the contents of the match, but I decided against doing that because it can get rather cumbersome to read in a table format.